src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs

```diff
@@ -1,27 +1,24 @@
 ﻿using System;
 using System.Windows;
 using System.Windows.Controls;
-using System.Windows.Data;
 using System.Windows.Input;
 using System.Windows.Media;
 using System.Windows.Media.Imaging;
 
 using Dynamo.Controls;
 using Dynamo.Logging;
-using Dynamo.Selection;
 using Dynamo.Utilities;
 using Dynamo.ViewModels;
 using Dynamo.Wpf.Utilities;
 
 using TextBox = System.Windows.Controls.TextBox;
-using UserControl = System.Windows.Controls.UserControl;
 
 namespace Dynamo.Search
 {
     /// <summary>
     ///     Interaction logic for SearchView.xaml
     /// </summary>
-    public partial class SearchView : UserControl
```

Comment:
it is, but is is partial class and base class is already specified where the other part of class is located. so specifying base class is redundant here